reactor-core/src/main/java/reactor/core/scheduler/Schedulers.java

```diff
@@ -1044,11 +1044,12 @@ public boolean isDisposed() {
 
 		@Override
 		public String toString() {
-			return cached.toString();
+			return "Schedulers." + key + "()";
```

Comment:
make sure that this is not a "hot" method (e.g. being frequently called from Micrometer or something), otherwise we may want to "cache" this string since it is immutable

Reply:
good point